nginx.conf

```diff
@@ -26,18 +26,37 @@ server {
     location / {
         include uwsgi_params;
         uwsgi_pass unix:///tmp/kolibri_uwsgi.sock;
-        error_page 502 = @502;
+        error_page 400 401 402 403 404 405 406 407 408 409 410 411 412 413 414 415 416 417 418 421 422 423 424 426 428 429 431 451 500 501 502 503 504 505 506 507 508 510 511 @error500;
     }
+    location @error500 {
+        ssi on;
+        internal;
+        root /etc/kolibri/dist/error_pages;
+
+
+        set $error500 /error.html;
+        # Spanish (Latinamerica)
+        if ($request_uri ~ "^\/es-419\/") {
+            set $error500 /la/error.html;
+        }
```

Comment:
Have you ever read ["If is evil"](https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/) regarding nginx? I highly suggest using the `map` directive for all of these `if` statements http://nginx.org/en/docs/http/ngx_http_map_module.html

Reply:
@bjester thanks so much for this suggestion. I was not aware of such evilness. I changed it to use map, also the config looks much better now.